addons-l10n/en/blocks2image.json

```
@@ -1,7 +1,8 @@
 {
-  "blocks2image/export_selected_to_SVG": "Export block as SVG",
-  "blocks2image/export_all_to_SVG": "Export all as SVG",
-  "blocks2image/export_selected_to_PNG": "Export block as PNG",
-  "blocks2image/export_all_to_PNG": "Export all as PNG",
-  "blocks2image/error_blocks_not_added": "Add blocks to workspace!"
+  "blocks2image/saveAll": "Save all as an image",
+  "blocks2image/save": "Save as an image",
+  "blocks2image/modalTitle": "Blocks to image",
+  "blocks2image/clipboard": "Copy to clipboard",
+  "blocks2image/svg": "Download as a SVG",
+  "blocks2image/png": "Download as a PNG"
```

Sounds weird to say "a SVG", I may change it...

```suggestion
  "blocks2image/svg": "Download as SVG",
  "blocks2image/png": "Download as PNG"
```

---

Would be more appropriate to do "an SVG" ("an es-vee-gee") grammar-wise, but this is better.